src/app/services/game-play.service.ts

```diff
@@ -145,6 +145,22 @@ export class GamePlayService {
     );
   }
```

Comment:
Actually, I think `take(1)` will cause issues because each keyboard character won't get updated properly then.

Maybe it's more of a stylistic thing, but I think it'd make more sense to put this observable in `key.component.ts` and expose it as a property.

Reply:
I did add `take(1)`! But it was removed in the Change Key Colors PR because at that point removing it seemed to be the only way to get the keys to update after a new guess had been submitted. But hindsight being 20/20, I don't believe that was an issue with the `take(1)` but rather it was an issue with how I was calling evaluateKey$ in the component/setting the class. Since I refactored per Sophia's recommendation, that issue seems to have been resolved so I'll add it back in!

Reply:
> Question: Is GamePlayService where evaluateKey$ should live? Or should evaluation take place in [KeyComponent](https://github.com/kelseyroy/wangle/blob/keyboard/src/app/key/key.component.ts) (see PR https://github.com/kelseyroy/wangle/pull/9 for context)?

I was definitely already considering moving evaluateKey (and by extension, evaluateLetter) into the Key and BoardTile components, respectively. I think that makes sense so I'll go ahead and execute that as well.

Reply:
What about `distinctUntilChanged()`? `take(1)` is still preventing any updates to the classes once new guesses are added, but `distinctUntilChanged()` seems to be fine. If I understand it correctly, it should prevent the observable from emitting again unless the emitted value is different from the last, correct?